bHY6yh-d30103/cod-bqe7s7_bHY6yh-d30103.xml

```diff
@@ -0,0 +1,491 @@
+               <lb ed="#P"/>Veteris amicitie aliqua reuerentia semper 
+               <lb ed="#P"/>impendenda est / nisi eam enormitas rea<lb ed="#P" break="no"/>tus 
+               eximeret.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e913">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e1029">Gregorius in pastorali capitulo xlvii</ref> <quote xml:id="bHY6yh-d30103-Qd1e917">malorum 
+               <lb ed="#P"/>cum incaute amicitiis iungimur cul<lb ed="#P" break="no"/>pis 
+               alligamur.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e923">
+               <cb ed="#P" n="b"/>
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e1041">Versificator fabularum exopi liiii fabula</ref> 
+               <lb ed="#P"/><quote xml:id="bHY6yh-d30103-Qd1e927">Nemo sibi satis est / eget omnis amicus amico.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e937">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e1057">Auctor libri Qui dicitur facetus capitulo vi</ref> 
+               <lb ed="#P"/><quote xml:id="bHY6yh-d30103-Qd1e941">Vtilius homini nichil est quam fidus amicus
+               <lb ed="#P"/>Vt ueluti secum cuncta loquatur ei
+               <lb ed="#P"/>Rebus in aduersis dabit hic solatia faci
+               <lb ed="#P"/>Prosperitate quidem gaudet utrique magis
+               <lb ed="#P"/>Prouidus eiusdem doctrine quirat amandio</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e955">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e1077">Gualterius de castellione libro vi</ref> 
+               <lb ed="#P"/><quote xml:id="bHY6yh-d30103-Qd1e959">Quem fastidit homo non vere diligit illo
+               <lb ed="#P"/>Verus amor miserum nunquam fastidit amicum.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e967">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e1091">Mattheus vindocinensis de istoria tobbie
+               <lb ed="#P"/>capitulo xxxii</ref> <quote xml:id="bHY6yh-d30103-Qd1e971">Est sine melle fauus sine 
+               <lb ed="#P"/>messe seges sine fructu. Arbor amici<lb ed="#P" break="no"/>tie 
+               coniuge vita carens.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e981">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e1108">Iacobus beneuentanus capitulo xxxiiii</ref> 
+               <lb ed="#P"/><quote xml:id="bHY6yh-d30103-Qd1e985">Omnibus est rebus bonus anteferendus amicus
+               <lb ed="#P"/>In verbis fidus non delectatur amicus.
+               <lb ed="#P"/>Gaudet enim factis commoda ferre sibi
+               <lb ed="#P"/>Infidum fugias fidumque sequaris amicum
+               <lb ed="#P"/>huic proposse stude ferre benignus opem</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e1003">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e1132">Auctor libri Qui dicitor astrolabi:</ref> 
```

Comment:
same comment here, colon punctuation should be inserted outside the closing ref tag.
```suggestion
               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e1132">Auctor libri Qui dicitor astrolabi</ref>:
```